client/src/components/JobInformation/JobInformation.vue

```diff
@@ -59,7 +59,11 @@ function filterMetadata(jobMessages) {
 
 async function fetchInvocation(jobId) {
     if (jobId) {
-        const invocation = await invocationForJob({ jobId: jobId });
+        const { data: invocation } = await GalaxyApi().GET("/api/invocations", {
+            params: {
+                query: { job_id: jobId },
+            },
+        });
```

Comment:
We're missing some kind of error handling here, no ?

Reply:
Well spotted I'll add at least a rethrowSimple here :+1:

We can follow up with more improved error handling in general looking for all the rethrowSimple calls :sweat_smile:

Reply:
Done! https://github.com/galaxyproject/galaxy/pull/18532/commits/37dcdf4a58cae25820bdb038e10dd349f8f2cf92